### PR TITLE
Try using ViteJS vor serve and build

### PR DESCRIPTION
**Pull request by tordans**
I was watching https://youtu.be/UJypSr8IhKY and thought of this project. ViteJS is said to be pretty fast and supports TailwindCSS and other PostCSS things out of the box. 

So it should solve https://github.com/pietervdvn/MapComplete/issues/201. And since that ticket seems to require updating parcel, maybe moving to ViteJS is even easier?

- https://vitejs.dev/
- https://vitejs.dev/guide/#command-line-interface
- https://github.com/vitejs/vite

## `npm run dev`

With the changes in this PR, I can start the dev server and everything looks normal. Did not test a lot, though.

## `npm run build-preview`

…which https://vitejs.dev/guide/#command-line-interface calls `serve`.

Start the server in the dist folder – so this depends on the build comman.

## `npm run build`

If I keep the purge list empty like https://github.com/tordans/MapComplete/blob/2302628a99083485a0c3e016d7aac9c0c4cc38fa/tailwind.config.js#L4-L8, the script runs until the end.
Otherwise it just stops doing stuff at some point – no error, no progress; I canceled after a while. No idea whats going on.

However, the dist folder does not include all the images and such. So when I run `npm run build-serve` the app works, but without the images. Maybe the `generate` task does not work or is in conflict with ViteJS?

## Takeaway

It still might be a good idea to use ViteJS instead of Parcel2. But with my pretty limited knowledge of the things I am talking about here I was not able to get it running.

--------

There is one other information, that might be interesting even without using ViteJS:

```
09:24:02 [vite] warning: Comparison using the "===" operator here is always false
42 |                  }
43 |
44 |                  if (json.elements === [] && json.remarks.indexOf("runtime error") > 0) {
   |                                    ^
45 |                      console.log("Timeout or other runtime error");
46 |                      onFail("Runtime error (timeout)")
```

### CODE REVIEW

**Comment by pietervdvn** [comment]
Hey Tobias,

Just a small heads up - I'm a bit overworked these days, I'll (try to) have a look to your PR closely next week.

**Comment by pietervdvn** [comment]
Hey Tobias,

Thanks for testing this!

However, I'm not convinced that using ViteJS will be worth it. It feels like a relatively young and unstable project; and switching seems to be a lot of work, without too much benefit except for a small performance improvement. I do not have the time available to figure out all this stuff (and, to be fair, I do not like to fight with this kind of systems to get them working).

In conclusion, I do not think that switching to ViteJS at this point is time is a good idea - but I do appreciate the effort you put in!
